docs/swagger.yaml

```diff
@@ -3241,3 +3538,44 @@ definitions:
       update_time:
         type: string
         description: The update time of label.
+  ProjectMember:
+    type: object
+    properties:
+      entity_id: 
+        type: integer
+        description: The ID of the member
+      entity_type: 
+        type: string
+        description: The type of the entity, u for user, g for group member
+      role_id: 
```

Comment:
Why call it `role_id` instead of `role` ?  Is it to avoid conflict with reserved word?